src/components/grids/DataGrid/PaginationControls.tsx

```diff
@@ -57,7 +57,7 @@ export default function PaginationControls({
         loading && { opacity: 0.5, pointerEvents: 'none' },
       ]}
     >
-      <div css={{ marginBottom: 10, display: 'flex', alignItems: 'center' }}>
+      <div css={{ marginBottom: 10, display: 'flex', alignItems: 'center' }} className="pagination-controls">
```

Comment:
out of curiosity, who sets css for the `pagination-controls` class? Is it web-eda? Or is the class name used another way?

Reply:
This is only used by web-eda currently, and is a handle for client packages to have finer control over CSS.